vulntotal/datasources/vulnerablecode.py

```diff
@@ -94,13 +121,13 @@ class VCIOTokenError(Exception):
 def fetch_vulnerablecode_query(url: str, payload: dict):
     """
     Requires VCIO API key in .env file
-    For example::
-
-              VCIO_TOKEN="OJ78Os2IPfM80hqVT2ek+1QnrTKvsX1HdOMABq3pmQd"
+    For example:
+        VCIO_TOKEN='OJ78Os2IPfM80hqVT2ek+1QnrTKvsX1HdOMABq3pmQd'
     """
+
     load_dotenv()
     vcio_token = os.environ.get("VCIO_TOKEN", None)
-    if not vcio_token:
+    if vcio_token is None:
```

Comment:
This will not handle the case when vcio_token is empty string.

```python
>>> vcio_token = ""
>>> vcio_token is None
False
```

```suggestion
    if not vcio_token:
```